client/src/connection/rest/index.ts

```diff
@@ -39,6 +39,36 @@ class RestSession extends Session {
     this._config = value;
   }
 
+  public async contextAttributes(): Promise<SessionContextAttributes> {
+    const context = await this.getContext();
+    // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+    return context.attributes as SessionContextAttributes;
```

Comment:
nit: shall we cache the attributes when create session so that we don't need to send getContext HTTP request twice?